languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -3676,6 +3676,18 @@ though, and has since been slightly modified)-->
     </rulegroup>
     <rulegroup id="SUR_ACCENT" name="Sur au lieu de Sûr">
       <url>http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2232</url>
+      <rule default="temp_off">
```

Comment:
If you want to add a new subrule to a rulegroup, it's best if you add it at the very bottom. Otherwise, all the other subrule IDs are going to change (leading to unnecessary diff entries: `SUR_ACCENT[1] → SUR_ACCENT[2]`, etc.).